snippets/python/merge-dictionaries.py

```diff
@@ -0,0 +1,12 @@
+# Merge two dictionaries into a single dictionary.
+
+
+def merge_dictionaries(first, second):
+    first = first.copy()
+    first.update(second)
+    return first
+
+
+merge_dictionaries({"age": 25}, {"name": "Joe"})
+#
+# {'age': 25, 'name': 'Joe'}
```

Comment:
I think `{**first, **second}` is better, and is supported in 3.5+ as mentioned in the article, we only support Python 3.6 so this is ok to use.

Reply:
Cool. I have included both approaches now.